backend/controllers/quote.api.test.ts

```diff
@@ -75,4 +76,21 @@ describe('quote controller', () => {
 		expect(response.body.success).toEqual(false);
 		expect(response.body.message).toEqual('Something went wrong.');
 	});
+
+	it("/quotes/random should return a random quote", async () => {
```

Comment:
This test is not entirely valid IMO. It's setup to return just one fake quote so randomness is not checked here at all. Ideally you should mock QuoteModel in a way that it returns few quotes and then call the endpoint and check if it returned exactly one quote and if it's in the group returned by the mock.

Reply:
I made a new commit to add true randomness to the test, ensuring a different quote is selected each time.